wrappers/kn/src/nativeMain/kotlin/zxingcpp/ReaderOptions.kt

```diff
@@ -0,0 +1,125 @@
+package zxingcpp
+
+import cnames.structs.zxing_ReaderOptions
+import kotlinx.cinterop.CValuesRef
+import kotlinx.cinterop.ExperimentalForeignApi
+import zxingcpp.cinterop.*
+import zxingcpp.cinterop.zxing_Binarizer.*
+import zxingcpp.cinterop.zxing_EanAddOnSymbol.*
+import zxingcpp.cinterop.zxing_TextMode.*
+
+@OptIn(ExperimentalForeignApi::class)
+class ReaderOptions(
+    formats: Set<BarcodeFormat> = setOf(BarcodeFormat.Any),
```

Comment:
That should probably be an empty set. This is internally mapped to "any" automatically. It would just be more consistent with the other wrappers.

Reply:
As to my concern, reflecting this in the upper wrapper will be more intuitive, although their behaviour is identical.

Reply:
I thought there was another comment from me a few lines below realizing that having those stateful variables as a copy of the underlying c++ values is again more than minimal. If they are not required by the language to get the property accessors below, then I'd rather not have them. See my "minimal code" comment from earlier.

Reply:
Does the minimal code mentioned here mean letting the underlying cpp library control its behavior as much as possible?
Hope I got you

Reply:
fixed in: https://github.com/zxing-cpp/zxing-cpp/pull/719/commits/6d1c50c8bec948d2126230c0f3713d15765f00fd